infra/main.tf

```diff
@@ -36,18 +36,16 @@ locals {
     host-group      = "buildpipeline"
     infra-owner     = "daml-language"
     managed         = "true"
-
-    # default the target name to be the name of the folder
-    target = "${basename(path.module)}"
+    target          = "infra"
```

Comment:
`basename(path.module)` somehow changed from `infra` to `.` in this new version. I could not find the documentation for `path.module`, and the expression `basename(path.module)` still produces `infra` in the `terraform console`, so I gave up.